doc/ideas/gameplay.md

```diff
@@ -135,6 +230,16 @@ Unit handling
  - Non-coward stance for villagers
  - Allow selecting the default stance
 
+### Viewcone-feature for additional complexity
+
+ - a viewcone shows in which direction the unit is looking at and how much it can see from that
+   - more complex Line-of-sight that this 360°-LOS
```

Comment:
Line-of-sight --> line of sight
that this -->than

Reply:
done